src/holidays/united-states/federal-reserve-bank/add-federal-reserve-bank-business-days.ts

```diff
@@ -0,0 +1,33 @@
+// holidays/united-states/federal-reserve-bank/add-federal-reserve-bank-business-days.ts
+
+import isFederalReserveBankHoliday from './is-federal-reserve-bank-holiday';
+import type { PlainDate } from './plain-date';
+
+export default function (plainDate: PlainDate, amount: number): PlainDate {
+  if (Number.isNaN(amount)) {
+    throw new TypeError('Invalid Amount');
+  }
```

Comment:
maybe this should also check for negative numbers? it looks like it would just be the same as sending 0.

Reply:
good point @bobwieler. But, not sure about a negative number is the same as sending 0. Technically, a negative number here should do what `subtract-federal-reserve-bank-business-days.ts` does, but `subtract-federal-reserve-bank-business-days.ts` should not accept negative numbers. I guess, the best option is to not allow negative numbers for both of these functions. what do you think?

Reply:
you could either remove subtract-federal-reserve-bank-business-days and allow negative numbers on an add to do the same thing or you could throw an error if someone decides to pass in a negative number. i kind of like the former as it's less code and seems to match what other apis do with negative numbers.